src/pages/detail/PlaceDetailPage.jsx

```
@@ -1,346 +1,173 @@
-        Accept: 'application/json',
-        ...(accessToken && { Authorization: `Bearer ${accessToken}` }),
-      };
-  
+
+      // 장소의 찜 업데이트 엔드포인트 (예시)
       await axios.post(
-        `https://meong9.store/api/v1/pl/likes/${id}`,
+        `https://meong9.store/api/v1/places/likes/${id}`,
         {},
-        { headers },
+        { headers: { Authorization: `Bearer ${accessToken}` } }
```

ESLint

[error] Insert `,`

(prettier/prettier)